daemon/cmd/daemon.go

```diff
@@ -303,7 +301,7 @@ func (d *Daemon) init() error {
 // createPrefixLengthCounter wraps around the counter library, providing
 // references to prefix lengths that will always be present.
 func createPrefixLengthCounter() *counter.PrefixLengthCounter {
```

Comment:
Incidentally `createPrefixLengthCounter()` and friends is also a legacy ipcache hmap handling function, we could look at removing that too. But we can follow up on that separately if that's easier. It's logically different from creating a cell for the ipcache initialization.